website/content/stocks/options/binom/_index.md

```diff
@@ -0,0 +1,17 @@
+```
+usage: binom [-s STRIKE] [-p] [-e] [-E] [-P] [-v VOLATILITY] [-h]
+```
+
+Shows the value of an option using binomial option pricing. Can also show raw data and provide a graph with predicted underlying asset ending values. The binomial options model calculates how big an up step or down step in the next time period will likely be. Then it creates a tree doing this at each period. The end results of this is a tree with possible asset values at each "step". For our calculations we use a day as our "step" time period. We then take all of the expected values at the finishing date and use this to begin a tree of option values at each step. The ending results is the value of the option today.
```

Comment:
Provide more detail about the calculation for the probability of up vs down and whatnot.

Reply:
Improved description